From 1af8509d6dd3d9ab97031426c6e78473b81ee799 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Fri, 10 Mar 2017 10:11:16 +0100
Subject: [PATCH 1424/1566] net: cadence: macb: Fix kernel-doc format

commit  6cbc5cde6a3f3daab80cd02d9c15571b3da85e3a from
https://github.com/Xilinx/linux-xlnx.git

Warnings:
drivers/net/ethernet/cadence/macb.c:268: info: Scanning doc for
drivers/net/ethernet/cadence/macb.c:274: warning: No description found
for parameter 'clk'
drivers/net/ethernet/cadence/macb.c:274: warning: No description found
for parameter 'speed'
drivers/net/ethernet/cadence/macb.c:274: warning: No description found
for parameter 'dev'

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/ethernet/cadence/macb.c |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/net/ethernet/cadence/macb.c b/drivers/net/ethernet/cadence/macb.c
index ca9735b..4326295 100644
--- a/drivers/net/ethernet/cadence/macb.c
+++ b/drivers/net/ethernet/cadence/macb.c
@@ -268,10 +268,10 @@ static int macb_mdio_write(struct mii_bus *bus, int mii_id, int regnum,
 }
 
 /**
- * macb_set_tx_clk() - Set a clock to a new frequency
- * @clk		Pointer to the clock to change
- * @rate	New frequency in Hz
- * @dev		Pointer to the struct net_device
+ * macb_set_tx_clk - Set a clock to a new frequency
+ * @clk:	Pointer to the clock to change
+ * @speed:	New frequency in Hz
+ * @dev:	Pointer to the struct net_device
  */
 static void macb_set_tx_clk(struct clk *clk, int speed, struct net_device *dev)
 {
-- 
1.7.5.4

